server/api/artworks.js

```diff
@@ -22,7 +22,12 @@ router.post('/', async (req, res, next) => {
   if (req.user && req.user.isAdmin) {
     try {
       const art = validateData(req.body, artProperties)
-      const newArt = await Artwork.create(art, { include: [Artist] })
+
+      //temp fix for new art having no genre
+      //artwork needs an array of genres for some frontend mapping
+      art.genres=[]
+
```

Comment:
thanks for catching this! once everything else is solidified i'll return to the task of figuring out how arrays into checkboxes into react local state works.